terrain/chunk.cc

```diff
@@ -0,0 +1,238 @@
+#include <vector>
+#include <math.h>
+#include <stdio.h>
+
+#include "chunk.h"
+#include "perlin.h"
+#include "mc.h"
+
+namespace Terrain {
+
+	float isoLevel = 0.0;
+
+	Chunk::Chunk(float origin[3], float chunkSize, int level, float x, float z) {
+		this->origin = {origin[0], origin[1], origin[2]};
+		this->unitSize = chunkSize / (float)this->segment;
+		this->points.resize((int)pow(this->segment + 1, 3));
+		this->max = {-10000.0, -10000.0, -10000.0, -10000.0};
```

Comment:
This should be infinity, which is a valid `float`.

Reply:
Sorry, @avaer. I haven't cleaned up old codes. We just need `terrain.cc` now